### PR TITLE
Hand Winner Function

### PR DESCRIPTION
**Pull request by nikhilgupta30**

### CODE REVIEW

File: src/lib/cardUtilFunc.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    // method to compare 2 cards
+    // card1 > card2 => 1   card2 > card1 => -1   card1=card2 => 0
+    public static int cardComparator(String card1, String card2, char trump) {
+
+        if(card1.equalsIgnoreCase(card2)){
+            return 0;
+        }
+
+        char suit1 = card1.charAt(1), suit2 = card2.charAt(1);
```
Comment (code-asdf) [inline src/lib/cardUtilFunc.java]: declare variables in a different line

Comment (code-asdf) [inline src/lib/cardUtilFunc.java]: Use utils in line 13

File: src/lib/cardUtilFunc.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            else return -1;
+        }
+        else if(suit1 == trump){
+            return 1;
+        }
+        else if(suit2 == trump){
+            return -1;
+        }
+        else{
+            return 1;
```
Comment (code-asdf) [inline src/lib/cardUtilFunc.java]: shouldn't this be -1?

File: src/lib/cardUtilFunc.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package lib;
+
+public class cardUtilFunc {
+
+    // method to compare 2 cards
+    // card1 > card2 => 1   card2 > card1 => -1   card1=card2 => 0
+    public static int cardComparator(String card1, String card2, char trump) {
```
Comment (code-asdf) [inline src/lib/cardUtilFunc.java]: boolean hi kr deta?

File: src/lib/cardUtilFunc.java (hunk 1 above)
Comment (nikhilgupta30) [reply src/lib/cardUtilFunc.java]: which constant in util can be used in line 13 ?

File: src/lib/cardUtilFunc.java (hunk 2 above)
Comment (nikhilgupta30) [reply src/lib/cardUtilFunc.java]: sahi pakde ho

File: src/lib/cardUtilFunc.java (hunk 3 above)
Comment (nikhilgupta30) [reply src/lib/cardUtilFunc.java]: kar diya

File: src/lib/cardUtilFunc.java (hunk 1 above)
Comment (code-asdf) [reply src/lib/cardUtilFunc.java]: utils.suit_index

**Review by code-asdf — APPROVED** [review]

**Pull request merged**